Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -99,7 +99,7 @@ extension SPAppDelegate {
             return false
         }
         popToNoteList()
-        noteListViewController.open(note, ignoringSearchQuery: true, animated: true)
+        presentNote(note)
```

Comment:
Not passing `ignoringSearchQuery: true ` changes current behaviour to reset search when opening a note.

Suggestion:
```
noteListViewController.open(note, ignoringSearchQuery: true, animated: false) 
```

Reply:
Thanks for the suggestion.  That works to use a fade transition as well.  I have updated the code.